flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -456,6 +468,59 @@ private Operation convertAlterDatabase(SqlAlterDatabase sqlAlterDatabase) {
 		return new AlterDatabaseOperation(catalogName, databaseName, catalogDatabase);
 	}
 
+	/** Convert CREATE VIEW statement. */
+	private Operation convertCreateView(SqlCreateView sqlCreateView) {
+		final SqlNode query = sqlCreateView.getQuery();
+		final SqlNodeList fieldList = sqlCreateView.getFieldList();
+
+		SqlNode validateQuery = flinkPlanner.validate(query);
+		PlannerQueryOperation operation = toQueryOperation(flinkPlanner, validateQuery);
+		TableSchema schema = operation.getTableSchema();
+
+		if (!fieldList.getList().isEmpty()) {
```

Comment:
fieldList could be empty ?

Reply:
yes. the fieldList is optional.

Reply:
I think it's better add some comments here or add an example, I doubt this because most people are  familiar with "create view ..." without alias.